hpi-archetype/pom.xml

```diff
@@ -40,18 +58,6 @@
     <url>http://github.com/jenkinsci/${project.artifactId}-plugin</url>
   </scm>
    -->
-  <repositories>
```

Comment:
An if deployed to central, the parent pom should not reference other repos, ergo, let's move this back to the archetype and to the tested plugins.

:+1: